qa/tasks/cephfs/test_damage.py

```diff
@@ -185,6 +185,8 @@ def __hash__(self):
                 # The PurgeQueue is allowed to be empty: Journaler interprets
                 # an empty header object as an empty journal.
                 expectation = NO_DAMAGE
+            elif o == "1.00000000" or re.search('[0-9]{8,}.[0-9]{8,}', o):
+                expectation = NO_DAMAGE
```

Comment:
'data_objects' contains objects whose size > 0. dirfrag object's size is 0. 'data_objects' should not contain dirfrag objects.

Reply:
You're right, I'll remove that.